onnxruntime/core/session/inference_session.cc

```diff
@@ -451,7 +464,7 @@ common::Status InferenceSession::SaveToOrtFormat(const std::basic_string<ORTCHAR
   fbs_buffer_size = ((fbs_buffer_size + m_bytes - 1) / m_bytes) * m_bytes;
   flatbuffers::FlatBufferBuilder builder(fbs_buffer_size);
 
-  auto ort_version = builder.CreateString(ORT_VERSION);
+  auto ort_version = builder.CreateString(s_ort_model_version);
   flatbuffers::Offset<fbs::Model> model;
```

Comment:
This should use ORT_VERSION shouldn't it? It will always be the current version.